File: Question5/Question5.java
import java.io.*;
import java.util.*;

class Solution {
    static int solution(int cutoffScore, int scoresLength, List<Integer> scores) {
        // Write solution here
        return 0;
    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int cutoffScore = Integer.parseInt(br.readLine());

        int scoresLength = Integer.parseInt(br.readLine());

        String str = br.readLine();
        String[] price_str = str.split(" ");

        List<Integer> scores = new ArrayList<Integer>();
        for (String s : price_str) {
            scores.add(Integer.parseInt(s));
        }

        int answer = solution(cutoffScore, scoresLength, scores);
        System.out.println(answer);
    }
}

File: Question5/Question5.js
'use strict';

process.stdin.resume();
process.stdin.setEncoding('utf-8');

let inputString = '';
let currentLine = 0;

process.stdin.on('data', inputStdin => {
    inputString += inputStdin;
});

process.stdin.on('end', _ => {
    inputString = inputString.replace(/\s*$/, '')
        .split('\n')
        .map(str => str.replace(/\s*$/, ''));

    main();
});

function readLine() {
    return inputString[currentLine++];
}

function solution(cutoffScore, scoresLength, scores) {
    // Write solution here
    return 0;
}

function main() {
    var cutoffScore = parseInt(readLine());

    var scoresLength = parseInt(readLine());

    var scores = readLine().split(' ').map(Number);

    var answer = solution(cutoffScore, scoresLength, scores);
    console.log(answer);
}
